src/javascript/app/pages/cashier/cashier.js

```diff
@@ -219,6 +219,12 @@ const Cashier = (() => {
                 const currency   = Client.get('currency');
                 if (is_virtual) {
                     displayResetButton();
+                    $('.change-account-btn-VRTC').off('click').on('click', (e)=>{
+                        e.stopPropagation();
+                        $('.main-account').trigger('click');
+                        $('html, body').animate({ scrollTop: 0 }, 'slow');
```

Comment:
in other parts of the code we are using something like `$.scrollTo('body', 500);`, can we use the same here?

Reply:
sure